.dockerignore

```diff
@@ -1,6 +1,7 @@
 **/\.*
 !.git
 !**/.eslintrc
+**/*.test.yml
```

Comment:
fuzz?

Reply:
This is since building a docker image doesn't require the compsoe file in the build context.